test/test_mllm.py

```diff
@@ -80,8 +80,19 @@ def test_quant_block_names(self):
         blocks = find_matching_blocks(model, all_blocks, to_quant_block_names)
         assert target_blocks == blocks
         
+    def test_dataset_check(self):
+        from auto_round.mllm.mllm_dataset import MLLM_DATASET
+        class Myclass:
+            model_type=None
+        dataset = MLLM_DATASET['liuhaotian/llava'](template=Myclass(), model=None, tokenzier=None, dataset_path="liuhaotian/llava", seqlen=32, nsamples=32)
+        self.assertEqual(len(dataset.questions), 32)
+        dataset = MLLM_DATASET['liuhaotian/llava'](template=Myclass(), model=None, tokenzier=None, dataset_path="liuhaotian/llava", seqlen=2048, nsamples=512)
```

Comment:
@n1ck-guo  you may need to have a check of this test on your pr

Reply:
i will work on this pr, cause the new push of weiwei retriggered the ut.